backend/tournesol/utils/video_id.py

```diff
@@ -0,0 +1,6 @@
+import random
+import string
+
+
+def generate_youtube_id():
+    return ''.join(random.choices(string.ascii_uppercase + string.digits, k=11))
```

Comment:
This functional will probably be used in tests only, so I would define it directly in the factory.